docs/cimpl6Tutorial_helloWorld.md

```diff
@@ -149,10 +149,19 @@ cd ~/cimpl/shr-cli
 yarn run ig:publish
 ```
 
-The generated profile page will be located at `~/cimpl/shr-cli/out/fhir/guide/output/StructureDefinition-hello-HelloWorld.html`.  Open this page in your favorite web browser.
+The generated **profile** page will be located at _~/cimpl/shr-cli/out/fhir/guide/output/StructureDefinition-hello-HelloWorld.html_.  Open this page in your favorite web browser.
 
-You should see a FHIR profile for `HelloWorld` in a (mostly empty) [FHIR Implementation Guide](https://www.hl7.org/fhir/implementationguide.html). 
+You should see a FHIR **profile** for _HelloWorld_ in a (mostly empty) [FHIR Implementation Guide](https://www.hl7.org/fhir/implementationguide.html). 
 
-**You did it!**
+_You did it!_
 
 For a more comprehensive understanding of the CIMPL grammar and how to use it for your project, continue to the [detailed tutorial](cimpl6Tutorial_detail.md).
+
+# Appendix A - Document Conventions
+
+| What you see | Explanation |
+|:----------|:---------|
+| Bolded Text  | FHIR reference, first letter in what will become an acronym
```

Comment:
The more I see this, the more I dislike it. I don't mind highlighting the first occurrence of a word like "profile", but I can't see any benefit from bolding it over and over.  Just getting antsy about this choice.